cmd/podman/buildfarm/inspect.go

```diff
@@ -0,0 +1,40 @@
+package buildfarm
+
+import (
+	"github.com/containers/podman/v4/cmd/podman/common"
+	"github.com/containers/podman/v4/cmd/podman/registry"
+	"github.com/spf13/cobra"
+)
+
+var (
+	buildfarmInspectDescription = `Get platform and status data from nodes in a farm`
+
+	inspectCommand = &cobra.Command{
+		Use:               "inspect [options] [FARM...]",
+		Short:             "Get platform and status data from nodes in a farm",
+		Long:              buildfarmInspectDescription,
+		RunE:              inspect,
+		ValidArgsFunction: common.AutoCompleteFarms,
+		Example: `podman buildfarm inspect myfarm
+  podman buildfarm inspect --all`,
+	}
+
+	// Temporary struct to hold cli values.
+	inspectOpts = struct {
+		All bool
+	}{}
+)
+
+func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Command: inspectCommand,
+		Parent:  buildfarmCmd,
+	})
+	flags := inspectCommand.Flags()
+	flags.BoolVarP(&inspectOpts.All, "all", "a", false, "Inspect all farms")
```

Comment:
To keep things simple and consistent with other command, I would drop the `--all` flag.  A `list --format json` usually does this kind of job.

Reply:
fixed